testsupport/wait/host.go

```diff
@@ -770,24 +769,27 @@ func (a *HostAwaitility) WaitAndVerifyThatUserSignupIsNotCreated(t *testing.T, n
 }
 
 // WaitForBannedUser waits until there is a BannedUser available with the given email
```

Comment:
```suggestion
// WaitForBannedUser waits until there is a BannedUser available with the given email hash
```
we could update the comment since now it expects an email hash and not email if I'm seeing this correctly

Reply:
Yes, thanks! I missed!